src/main/sensors/gyro.c

```diff
@@ -635,17 +635,29 @@ void dynLpfGyroUpdate(float throttle)
         } else {
             cutoffFreq = fmax(dynThrottle(throttle) * gyro.dynLpfMax, gyro.dynLpfMin);
         }
-        if (gyro.dynLpfFilter == DYN_LPF_PT1) {
-            DEBUG_SET(DEBUG_DYN_LPF, 2, cutoffFreq);
-            const float gyroDt = gyro.targetLooptime * 1e-6f;
+        DEBUG_SET(DEBUG_DYN_LPF, 2, cutoffFreq);
+        const float gyroDt = gyro.targetLooptime * 1e-6f;
+        switch (gyro.dynLpfFilter) {
+        case DYN_LPF_PT1:
             for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
                 pt1FilterUpdateCutoff(&gyro.lowpassFilter[axis].pt1FilterState, pt1FilterGain(cutoffFreq, gyroDt));
             }
-        } else if (gyro.dynLpfFilter == DYN_LPF_BIQUAD) {
-            DEBUG_SET(DEBUG_DYN_LPF, 2, cutoffFreq);
+            break;
+        case DYN_LPF_BIQUAD:
+            for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
+            biquadFilterUpdateLPF(&gyro.lowpassFilter[axis].biquadFilterState, cutoffFreq, gyro.targetLooptime);
+            }
+            break;
+        case  DYN_LPF_PT2:
             for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-                biquadFilterUpdateLPF(&gyro.lowpassFilter[axis].biquadFilterState, cutoffFreq, gyro.targetLooptime);
+                pt2ilterUpdateCutoff(&gyro.lowpassFilter[axis].pt2FilterState, pt2ilterGain(cutoffFreq, gyroDt));
```

Comment:
Typos on the function names. Please make sure the code actually compiles before pushing a commit.

Reply:
There is no easy way in the code for any process or user action to set the cutoff anything close to Nyquist.  So the check is a safety feature in case the code fails or the loop time is set extremely low, both of which are most unlikely.
However, as we both well know, in either case the PTn code is unconditionally stable no matter what the proposed cutoff value is, since the way it was coded includes protection against excessively high set cutoffs.  
Furthermore, setting a very high cutoff, intentionally or otherwise, only has the result that output essentially equals input, which is what would be expected anyway.
So there is zero hazard and hence no need to waste CPU cycles on an unnecessary check.
The reason to remove the check is not because there is any virtue or purpose in setting it really high, or to permit people setting it high, for which we agree, there is no point.  The purpose is only to remove unnecessary code. 
Therefore I will leave the code as is.  
If you believe there is a usage case that justifies adding it later, then make a PR that solves a problem and justifies the extra code.